zipkin-api.yaml

```diff
@@ -248,17 +248,30 @@ definitions:
     properties:
       traceId:
         type: string
+        description: 'Randomly generated globally unique transaction/context id'
```

Comment:
In code we used `Unique 8-byte identifier for a trace, set on all spans within it.` Good point that it should be random.. but it is also hex-encoded.. I wouldn't conflate it for a transaction or context id. Also global is relative. Maybe..

`Randomly generated, unique 8-byte identifier for a trace, set on all spans within it. Encoded into 16 lowercase hex characters`